ansible_collections/arista/avd/roles/eos_designs/python_modules/underlay/router_bgp.py

```diff
@@ -125,6 +125,7 @@ def router_bgp(self) -> dict | None:
                     "peer": link["peer"],
                     "description": "_".join([link["peer"], link["peer_interface"]]),
                     "bfd": get(link, "bfd"),
+                    "shutdown": True if link["peer_is_deployed"] is False else None,
```

Comment:
This will change existing behavior. We need a global toggle similar to `shutdown_interfaces_towards_undeployed_peers` which should default to false and keep the existing behavior.
`shutdown_bgp_towards_undeployed_peers`

Reply:
Since the check you need will be long, I suggest to move it below in an `if`.

Reply:
Fixed.